src/azure-cli/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -60,3 +63,6 @@ def load_command_table(self, _):
     with self.command_group('servicebus namespace encryption', custom_command_type=sb_namespace_custom, is_preview=True) as g:
         g.custom_command('add', 'cli_add_encryption')
         g.custom_command('remove', 'cli_remove_encryption')
+
+# Migration Region
+    self.command_table['servicebus migration update'] = Update(loader=self, deprecate_info=self.deprecate())
```

Comment:
I don't see a new import, so this `Update` is the same with which was imported from line 43, right?
```
from azure.cli.command_modules.servicebus.aaz.latest.servicebus.georecovery_alias._update import Update
```

Reply:
Yeah, I forgot this is a different import.
Corrected now.
Thanks!